### DISCUSSION TITLE
How do I serialize and deserialize Entity?

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2518] [Q&A] [answered]
How do I serialize and deserialize Entity?

Currently I am implementing a caching method, but the MikroORM relationship is not recognized after deserialization

**Comment by B4nan** [comment, marked as answer]
Would be good if you add more context to your questions, you clearly have a very specific issue, but without any code its hard to guess which one.

```ts
// serialize regardless populate hints
const dto = wrap(entity).toPOJO();

// deserialize
const entity = em.merge(MyEntity, dto);
```

**Reply by Lydanne** [reply]
i trry

**Reply by Lydanne** [reply]
What is the difference toPOJO and toJSON ？

**Reply by Lydanne** [reply]
![image](https://user-images.githubusercontent.com/39021696/145790409-29e13e8f-8b70-48cb-bb6c-713262727b28.png)

**Reply by Lydanne** [reply]
I didn't test the difference
